packages/mobile/locales/en-US/onboarding.json

```diff
@@ -29,7 +29,7 @@
   "selectCountryCode": "Select Country Code",
   "pincodeSet": {
     "create": "Create a PIN",
-    "verify": "Enter PIN again",
+    "changePIN": "Change PIN",
```

Comment:
Is this change on purpose? The `verify` is used in the onboarding flow, we should probably not delete it :)

Reply:
oops! took it out thinking i had added it on accident-  thanks!